core/src/main/scala/caliban/parsing/parsers/StringParsers.scala

```diff
@@ -32,9 +32,9 @@ private[caliban] trait StringParsers {
         (state: @switch) match {
           case Normal                  =>
             (currentChar: @switch) match {
+              case CR                                    => loop(index + 1, state = DetermineLineBreakStart)
```

Comment:
This is a micro-optimization since new lines are very common in graphQL queries, and it seems to perform slightly better if we have this case on the top since it contains only 1 character (as opposed to 5 in the case below)